index.d.ts

```diff
@@ -70,6 +70,45 @@ declare namespace Eris {
   type PossiblyUncachedMessage = Message | { channel: TextableChannel | { id: string; guild?: Uncached }; guildID?: string; id: string };
   type InteractionType = 1 | 2;
 
+  //Slash Commands
+  type SlashCommandTypes = 1 | 2 | 3;
+
+  type SlashCommandOptionTypes = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
+
+  type SlashCommandOptions = {
+    type: SlashCommandOptionTypes;
+    name: string;
+    description: string;
+    required?: boolean;
+    choices?: { name: string; value: string | number};
+  };
+
+  type SlashCommand = {
+    id: string;
+    application_id: string;
+    guild_id?: string;
+    name: string;
+    description: string;
+    options?: SlashCommandOptions[];
+    type?: SlashCommandTypes;
+    defaultPermission?: boolean;
+  };
+
+  type SlashCommandStructure = Exclude<SlashCommand, "id" | "application_id" | "guild_id">;
```

Comment:
this should be `Omit` to remove the keys
```suggestion
  type SlashCommandStructure = Omit<SlashCommand, "id" | "application_id" | "guild_id">;
```